configure.ac

```diff
@@ -66,6 +66,12 @@ AC_ARG_ENABLE(settings,
               [enable_settings="yes"])
 AM_CONDITIONAL([BUILD_SETTINGS],[test "x$enable_settings" = "xyes"])
 
+AC_ARG_ENABLE([faces],
+              [AC_HELP_STRING([--enable-faces],
```

Comment:
This seems fine, though I do wonder whether it's necessary to have separate --disable-fonts / --disable-settings / --disable-faces when we either pass all or none. No harm though!